code/day1/day1_ACT_Node.R

```diff
@@ -8,7 +8,7 @@ library(viridis)
 library(plotly)
 library(ggmap)
 
-setwd('C:/Users/ct991305/Documents/Workshop Material/one true workshop - WIP/ACT_data') #set folder you're going to work in
+setwd('data/act') #set folder you're going to work in
```

Comment:
to match the format from migramar, should be changed to:

setwd('YOUR/PATH/TO/data/act')